### ISSUE TITLE
Contribute to the Group Report

### ISSUE THREAD

**Issue by eye-78** [bounswe/2021SpringGroup6#126]
- [x] Add summary of the project
- [x] Add documentation and code of your API
- [x] Describe functionality of the project
- [x] Comment on challenges faced
- [x] Contribute "Evaluation of Tools" part
- [x] Add personal effort
- [x] Add structure of the code and info about external APIs
- [x] Specify where your code is

**Comment by eye-78** [comment]
- Added summary of the project. It is deployed neary as it was.
- Added code and documentation of my API. Can be found in the document.
- Described the functionality of the project. @fileames added list of pages, other work belongs to me.
- Wrote first half of the part relatad to challenges we faced. @38programmer61 wrote the second half.
- Wrote evaluations of tools part with @berkatil. He initiated and built the srtucture and I finalized it.
- Added personal effort part. More than one page. Wow
- Added info about the code to the second document. It was used for specifying our efforts. 

Document is  [here](https://github.com/bounswe/2021SpringGroup6/blob/master/deliverables/M2_Report_Group6.pdf)
